Sources/Queues/Application+Queues.swift

```diff
@@ -21,14 +21,25 @@ extension Application {
         }
 
         final class Storage {
+            enum QueuesType: String {
+                case `default` = "queues"
+                case inProcessJob = "inProcessJobsQueues"
+                case scheduledJobs = "scheduledJobsQueues"
+            }
             public var configuration: QueuesConfiguration
-            let command: QueuesCommand
+            private (set) var commands: [QueuesType: QueuesCommand]
```

Comment:
Does this need to be a dictionary? Or can we just loop over the values and shut them down?

Reply:
The reason behind it was that I'm not sure about the purpose of three different `QueuesCommand` and whether we need a way to control/update each type. It definitely can be an array

Reply:
Gotcha, yeah let's leave it as an array